weasyl/test/login/test_verify.py

```diff
@@ -21,7 +21,7 @@ def _create_pending_account(invalid=False):
         "hashpass": login.passhash('0123456789'),
         "email": email,
         "birthday": arrow.Arrow(2000, 1, 1),
-        "unixtime": arrow.now(),
+        "created_at": arrow.now().datetime,
```

Comment:
It’s the `server_default`, so does removing it work?

```suggestion
```

Reply:
Yup, makes it a little cleaner!